submissions/asaMitaka/FriendsApp/script.js

```diff
@@ -0,0 +1,237 @@
+const url = 'https://randomuser.me/api/?results=12';
+const form = document.forms;
+const content = document.querySelector('.articleBlock');
+
+let friendsCopy = [];
+let friends = [];
+let isSearch = false;
+
+async function getResponse(url) {
+    let response;
+    try {
+        response = await fetch(url);
+        if (!response.ok) {
+            throw new Error(response.statusText);
+        } 
+    } catch (err) {
+        console.log(err);
+        resetPage();
+    }
+    let jsonData = await response.json();
+
+    friendsCopy = jsonData.results;
+    friends = jsonData.results;
+
+    renderAllItemsToPage(friends);
+}
+
+function creatingDivItem(item) {
```

Comment:
What `div` item? What is the content of this item? Use more descriptive names, please.